### PR TITLE
Updating logger to support different types

### LINKED ISSUE #445: [Scarpet Suggestion] Multiple log levels for Scarpet logger() function
Currently, the `logger()` function in Scarpet always uses the `error` log type, and there is no way to change it.
My suggestion would be an argument to specify the "severity" of the message to be logged, since currently the only way to change this is by using `print()` instead of `logger()`, which would use the regular `info` log type, but would also print the message in chat in most cases, and still `debug`, `warn` and `fatal` are unreachable.

(names based in the vanilla launcher's output log viewer)

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Fixes #445 

Im not sure whether it should output error by default, or I should hardcode at 2 by default, or maybe just set it to info by default. I think info makes more sense than warn...

### CODE REVIEW

**Comment by James103** [comment]
The commit "Fixes #437" should either be moved to its own PR or the title of the PR should be changed to account for the additional comment.

**Comment by gnembon** [comment]
Its fine as long as these tickets are closed. 

Please update docs as well for both changes though.

Also - default log level of 'info' is fine by me.

**Comment by Ghoulboy78** [comment]
ok, yeah forgot the thing aboot the docs

**Comment by Ghoulboy78** [comment]
btw, what are all the highlighted comments in the logger() setion about?

**Comment by Ghoulboy78** [comment]
Im not sure if those are the right colours tho.

File: docs/scarpet/api/Auxiliary.md [hunk 1]
```diff
@@ -245,13 +245,26 @@ Example usages:
...
 
-Prints the message to system logs, and not to chat.
+Prints the message to system logs, and not to chat. By default prints an info, unless you specify otherwise in the `type` parameter.
 
-### read_file(resource, type)
-### delete_file(resource, type)
-### write_file(resource, type, data, ...)
+Available output types:
+
+`'debug'`: white
```
Comment (gnembon) [inline docs/scarpet/api/Auxiliary.md:254]: there is no need mention colors - every system could have that configured differently.

You also forgot to document the 'air' entity API

Comment (gnembon) [inline docs/scarpet/api/Auxiliary.md:254]: there is no need mention colors - every system could have that configured differently.

You also forgot to document the 'air' entity API

Comment (Ghoulboy78) [reply docs/scarpet/api/Auxiliary.md:254]: ofc i did

**Comment by gnembon** [comment]
Forgot bout docs for `modify(e, 'air', ticks)`;P

**Comment by Ghoulboy78** [comment]
ofc I did

**Pull request merged**